snmp/tests/test_profiles.py

```diff
@@ -953,6 +969,35 @@ def test_aruba(aggregator):
         'snmp.wlsxSysExtPacketLossPercent', metric_type=aggregator.GAUGE, tags=common_tags, count=1
     )
 
+    # OSPF metrics
+    neighbor_metrics = [
+        ('ospfNbrEvents', aggregator.RATE),
+        ('ospfNbrState', aggregator.GAUGE),
+        ('ospfNbrLsRetransQLen', aggregator.GAUGE),
+    ]
+    for metric, metric_type in neighbor_metrics:
+        tags = ['neighbor_ip:192.29.116.26', 'neighbor_id:192.29.66.79'] + common_tags
+        aggregator.assert_metric('snmp.{}'.format(metric), metric_type=metric_type, tags=tags, count=1)
+
+    virtual_neighbor_metrics = [
+        ('ospfVirtNbrState', aggregator.GAUGE),
+        ('ospfVirtNbrEvents', aggregator.RATE),
+        ('ospfVirtNbrLsRetransQLen', aggregator.GAUGE),
+    ]
+    for metric, metric_type in virtual_neighbor_metrics:
+        tags = ['neighbor_ip:74.210.82.1', 'neighbor_id:203.164.233.193'] + common_tags
+        aggregator.assert_metric('snmp.{}'.format(metric), metric_type=metric_type, tags=tags, count=1)
+
+    lls_metrics = ['ospfIfRetransInterval', 'ospfIfState', 'ospfIfLsaCount']
+    for metric in lls_metrics:
+        tags = ['ospf_ip_addr:103.229.209.198'] + common_tags  # 'neighbor_ip:74.210.82.1',
```

Comment:
These tags were not found despite being asserted before. It could be something up with the .snmprec data and the fact that these are from a different table than the base metric.

Reply:
We need to fix the test data, looking at it.